### DISCUSSION TITLE
Nerdfont rendered weirdly on mac

### DISCUSSION THREAD

**Discussion by Tus1688** [kovidgoyal/kitty#4283] [Q&A]
Hi everyone, first of all thank you for this awesome terminal emulator. It's blazing fast... 

Nerdfont in iTerm2 renders like this:
<img width="149" alt="image" src="https://user-images.githubusercontent.com/76937231/143579923-009f35b9-57cc-4a4f-a7eb-72d1cac8c423.png">

Whereas in Kitty renders like this: 
<img width="135" alt="image" src="https://user-images.githubusercontent.com/76937231/143580124-b971c75e-8f3e-4ea7-9efe-22c84ec50327.png">

I use kitty on Linux and it rendered correctly, I used to spend couple of minute tweaking anti-aliasing between 0.2 -0.75. But it seems I can't solve this, does anyone has a clue? Thanks :)

"" <- this is the character

**Comment by kovidgoyal** [comment]
The first ball has no space after it, the second does. SO the second uses that space and renders larger. This is expected behavior.

**Reply by Tus1688** [reply]
It seems there is no solution for it?
